### PR TITLE
Electrictiy & Heat as products

### PR DESCRIPTION
**Pull request by dleard**
- Add heat/electricity related products
- Remove electricity & heat form from application wizard
- Remove old unused ciip_incentive_payment view
- Update all cypress tests after removal of electricity & heat form

* I have left the electiricty_and_heat.json file in, it's not being used anymore, but felt there was no reason do delete it entirely yet.
** I have also not removed the ciip_electricity_and_heat view as application_revision_ciip_incentive still requires it and it is being worked on concurrently with this PR. So no changes have been make to application_revision_ciip_incentive

### CODE REVIEW

File: schema/data/prod/product.sql [hunk 1]
```diff
@@ -39,7 +39,11 @@ values
 (33, 'Petroleum Refining','BC refining complexity factor unit','active',true),
 (34, 'Wood pellets', 'bone dry tonnes','active', true),
 (35, 'Wood chips', 'bone dry tonnes','active', true),
-(36, 'Waste rendering', 'tonnes','active', true)
+(36, 'Waste rendering', 'tonnes','active', true),
+(37, 'Imported electricity', 'MWh', 'active' ,true),
```
Comment (matthieu-foucault) [inline schema/data/prod/product.sql:43]:
```suggestion
(37, 'Purchased electricity', 'GWh', 'active' ,true),
```
Naming and units changes as requested by CAS

File: schema/data/prod/product.sql [hunk 2]
```diff
@@ -39,7 +39,11 @@ values
...
+(38, 'Imported heat', 'GJ', 'active' ,true),
```
Comment (matthieu-foucault) [inline schema/data/prod/product.sql:44]:
```suggestion
(38, 'Purchased heat', 'GJ', 'active' ,true),
```

File: schema/data/prod/product.sql [hunk 3]
```diff
@@ -39,7 +39,11 @@ values
...
+(39, 'Generated electricity', 'MWh', 'active' ,true),
```
Comment (matthieu-foucault) [inline schema/data/prod/product.sql:45]:
```suggestion
(39, 'Generated electricity', 'GWh', 'active' ,true),
```

File: schema/data/prod/product.sql [hunk 4]
```diff
@@ -39,7 +39,11 @@ values
...
+(40, 'Generated heat', 'GJ', 'active' ,true)
```
Comment (matthieu-foucault) [inline schema/data/prod/product.sql:46]: Add exported heat/electricity

File: schema/data/prod/form_json.sql [hunk 5]
```diff
@@ -10,9 +10,6 @@ create temporary table emission (json_data jsonb);
 create temporary table fuel (json_data jsonb);
 \copy fuel(json_data) from program 'sed ''s/\\/\\\\/g'' < prod/form_json/fuel.json | tr -d ''\n''';
 
-create temporary table electricity_and_heat (json_data jsonb);
-\copy electricity_and_heat(json_data) from program 'sed ''s/\\/\\\\/g'' < prod/form_json/electricity_and_heat.json | tr -d ''\n''';
```
Comment (matthieu-foucault) [inline schema/data/prod/form_json.sql:14]: the `prod/form_json/electricity_and_heat.json` file can be deleted now

File: schema/data/prod/product.sql (hunk 4 above)
Comment (dleard) [reply schema/data/prod/product.sql:46]: exported heat/electricity were already entered as products in the data/prod/products.sql file (id's 10 & 11)

Comment (matthieu-foucault) [reply schema/data/prod/product.sql:46]: Right, I didn't remember if I had done that or not

**Pull request merged**